roadmap.md

```diff
@@ -22,21 +22,26 @@ We must finish the Taskcluster migration. Supporting both Buildbot and
 Taskcluster within automation is a substantial drag on agility and
 productivity, and is costly in its resource consumption.
 
-### Sunsetting Deprecated Features
+The remaining bits of the migration include:
+ * Porting Windows tests to run natively in Taskcluster
+ * Porting Windows and Linux tasks that must run on hardware (Talos, etc.) to run natively in Taskcluster
```

Comment:
That seems like an implementation detail -- in this section I'm mostly trying to sketch out what's being tracked elsewhere.  But I'll re-word.